CONTRIBUTING.md

```diff
@@ -636,3 +732,63 @@ Each branch is [deployed](https://github.com/CrowdStrike/glide-core/blob/main/.g
 And artifacts from our latest publish are [deployed](https://github.com/CrowdStrike/glide-core/blob/main/.github/workflows/on-merge-branch-changeset-release-main.yml#L95) to the top level of the bucket.
 It's a simple setup.
 But it means we can't have branch names that conflict and thus overwrite top-level directories that contain artifacts.
+
+### How does ARIA snapshot testing fit into overall accessibility testing?
+
+We use Axe to test accessibility.
+But Axe only tells us that a component's accessibility tree doesn't contain any obvious violations and that the accessibility tree is valid.
+What Axe doesn't tell us is whether the accessibility tree is correct.
+
+Dropdown, for example, when open, should show up as expanded in the accessibility tree.
+But, if it doesn't, Axe won't throw because Dropdown's accessibility tree is valid when Dropdown is open but not expanded in the tree.
```

Comment:
```suggestion
But, if it doesn't, Axe won't throw because Dropdown's accessibility tree is valid when Dropdown is open even though it's not expanded in the tree.
```